test/smoke/smoke_test.go

```diff
@@ -58,6 +58,67 @@ var _ = Describe("Smoke test", func() {
 				return standalone.Status.Phase
 			}, deployment.GetTimeout(), PollInterval).Should(Equal(splcommon.PhaseReady))
 
+			Eventually(func() splcommon.Phase {
```

Comment:
Move this to a new test under ingest tests. We want to keep smoke_test.go strictly related to basic splunk deployments by kubernetes.

Reply:
Noted.  I'll move this test to the ingest_search_tests.

Reply:
Moved out of smoke to ingest_search_test